packages/cloudflare-workers-bindings-extension/src/extension.ts

```diff
@@ -1,9 +1,24 @@
 import * as vscode from "vscode";
+import { importWrangler } from "./wrangler";
 
 export async function activate(context: vscode.ExtensionContext) {
 	vscode.commands.registerCommand(
 		"cloudflare-workers-bindings-extension.testCommand",
 		() =>
 			vscode.window.showInformationMessage(`Successfully called test command.`)
 	);
+
+	const rootPath =
+		vscode.workspace.workspaceFolders &&
+		vscode.workspace.workspaceFolders.length > 0
+			? vscode.workspace.workspaceFolders[0].uri.fsPath
+			: undefined;
+
+	if (!rootPath) {
+		return;
+	}
+
+	const wrangler = importWrangler(rootPath);
```

Comment:
Nit: Should we make `importWrangler` accepts `undefined` so it could fallback to the bundled wrangler as well?

Reply:
I think the root path should always be defined (if it's not, there's no workspace loaded, and so there's nothing this extension can usefully do anyway)